Won't build with Node 4.1.0

```
npm install scrypt

> scrypt@5.1.0 install /home/rylor/buynothing/backend/node_modules/scrypt
> node-gyp rebuild

make: Entering directory '/home/rylor/buynothing/backend/node_modules/scrypt/build'
  COPY ../scrypt/scrypt-1.1.6/config.h
  SOLINK_MODULE(target) Release/obj.target/copied_files.node
  COPY Release/copied_files.node
  CC(target) Release/obj.target/scrypt_wrapper/src/util/salt.o
  CC(target) Release/obj.target/scrypt_wrapper/src/scryptwrapper/keyderivation.o
  CC(target) Release/obj.target/scrypt_wrapper/src/scryptwrapper/pickparams.o
  CC(target) Release/obj.target/scrypt_wrapper/src/scryptwrapper/hash.o
  AR(target) Release/obj.target/scrypt_wrapper.a
  COPY Release/scrypt_wrapper.a
  CC(target) Release/obj.target/scrypt_lib/scrypt/scrypt-1.1.6/lib/crypto/sha256.o
  CC(target) Release/obj.target/scrypt_lib/scrypt/scrypt-1.1.6/lib/crypto/crypto_scrypt-sse.o
  CC(target) Release/obj.target/scrypt_lib/scrypt/scrypt-1.1.6/lib/scryptenc/scryptenc_cpuperf.o
  CC(target) Release/obj.target/scrypt_lib/scrypt/scrypt-1.1.6/lib/util/memlimit.o
  CC(target) Release/obj.target/scrypt_lib/scrypt/scrypt-1.1.6/lib/scryptenc/scryptenc.o
  CC(target) Release/obj.target/scrypt_lib/scrypt/scrypt-1.1.6/lib/crypto/crypto_aesctr.o
  AR(target) Release/obj.target/scrypt_lib.a
  COPY Release/scrypt_lib.a
  CXX(target) Release/obj.target/scrypt/src/node-boilerplate/scrypt_common.o
../src/node-boilerplate/scrypt_common.cc:25:17: fatal error: nan.h: No such file or directory
compilation terminated.
scrypt.target.mk:108: recipe for target 'Release/obj.target/scrypt/src/node-boilerplate/scrypt_common.o' failed
make: *** [Release/obj.target/scrypt/src/node-boilerplate/scrypt_common.o] Error 1
make: Leaving directory '/home/rylor/buynothing/backend/node_modules/scrypt/build'
gyp ERR! build error 
gyp ERR! stack Error: `make` failed with exit code: 2
gyp ERR! stack     at ChildProcess.onExit (/usr/lib/node_modules/npm/node_modules/node-gyp/lib/build.js:270:23)
gyp ERR! stack     at emitTwo (events.js:87:13)
gyp ERR! stack     at ChildProcess.emit (events.js:172:7)
gyp ERR! stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:200:12)
gyp ERR! System Linux 4.1.6-1-ARCH
gyp ERR! command "/usr/bin/node" "/usr/lib/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js" "rebuild"
gyp ERR! cwd /home/rylor/buynothing/backend/node_modules/scrypt
gyp ERR! node -v v4.1.0
gyp ERR! node-gyp -v v3.0.1
gyp ERR! not ok 
npm ERR! Linux 4.1.6-1-ARCH
npm ERR! argv "/usr/bin/node" "/usr/bin/npm" "install" "scrypt"
npm ERR! node v4.1.0
npm ERR! npm  v3.3.3
npm ERR! code ELIFECYCLE

npm ERR! scrypt@5.1.0 install: `node-gyp rebuild`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the scrypt@5.1.0 install script 'node-gyp rebuild'.
npm ERR! This is most likely a problem with the scrypt package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     node-gyp rebuild
npm ERR! You can get their info via:
npm ERR!     npm owner ls scrypt
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     /home/rylor/buynothing/backend/npm-debug.log
```

Freshly installed with `npm install scrypt`

On Arch.

--- a/binding.gyp
+++ b/binding.gyp
@@ -151,7 +151,7 @@
       'scrypt_node.cc'
     ],
     'include_dirs': [
-      'node_modules/nan',
+      '<!(node -e "require(\'nan\')")',
       'src/util',
       'src/scryptwrapper/inc',
       'src/node-boilerplate/inc'

